pixeltable/tests/test_table.py

```diff
@@ -85,6 +85,23 @@ def test_create(self, test_client: pt.Client) -> None:
         with pytest.raises(exc.Error):
             cl.drop_table('.test2')
 
+    def test_attributes(self, test_client: pt.Client) -> None:
+        cl = test_client
+        schema = {'c': StringType(nullable=False)}
+        num_retained_versions = 20
+        description = "This is a table."
+        tbl = cl.create_table('test_attrs', schema, num_retained_versions=num_retained_versions, description=description)
+        assert(tbl.tbl_version.attributes.num_retained_versions == num_retained_versions)
+        assert(tbl.tbl_version.attributes.description == description)
+        new_num_retained_versions = 30
+        new_description = "This is an updated table."
+        tbl.tbl_version.update_attributes(num_retained_versions=new_num_retained_versions, description=new_description)
```

Comment:
TableVersion is not a user-facing class.

This should be done via Table, and we should probably call it something like set_attr or set_attribute (to keep with Python vernacular).

In a similar vein, it would make sense to add a Table.description() -> str.